<launch>
  <node
    pkg="skyrats_cbr_2021"
    type="cross_node"
    name="cross_node" 
    output="screen"
  />
  <node
    pkg="skyrats_cbr_2021"
    type="MRS_MAV.py"
    name="MRS_MAV"
    output="screen"
  />

  <node
  pkg="skyrats_cbr_2021"
  type="fase4.py"
  name="fase4"
  output="screen"
  />
</launch> 